src/routes/(app)/admin[[hash=admin_hash]]/arm/user/[id]/+page.server.ts

```diff
@@ -48,36 +48,25 @@ export const actions = {
 		if (user.roleId !== SUPER_ADMIN_ROLE_ID && !parsed.recoveryEmail && !parsed.recoveryNpub) {
 			throw error(400, 'You must provide a recovery email or npub');
 		}
-		if (user.roleId === SUPER_ADMIN_ROLE_ID) {
-			await collections.users.updateOne(
-				{ _id: user._id },
-				{
-					$set: {
-						...(parsedAdmin.alias && { alias: parsedAdmin.alias })
-					}
-				}
-			);
 
-			throw redirect(303, `${adminPrefix()}/arm`);
-		} else {
-			await collections.users.updateOne(
-				{ _id: user._id },
-				{
-					$set: {
-						login: parsed.login,
-						...(parsed.alias && { alias: parsed.alias }),
-						recovery: {
-							...(parsed.recoveryEmail && { email: parsed.recoveryEmail }),
-							...(parsed.recoveryNpub && { npub: parsed.recoveryNpub })
-						},
-						disabled: parsed.status === 'disabled',
-						roleId: parsed.roleId
-					}
+		await collections.users.updateOne(
+			{ _id: user._id },
+			{
+				$set: {
+					login: parsed.login ?? user.login,
```

Comment:
This allows anyone to update the super admin's login & recovery info, including non-super admins